index.html

```diff
@@ -41,8 +41,29 @@ <h2> Skills</h2>
 <section>
 <h1>Connect </h1>
 <a href= https://www.linkedin.com/in/kipardillo/> linkedin </a>
+</section>
+
+<section>
+    <h2> Leave a Message</h2>
+    <form name = "leave_message">
+        <label for  = leave_message> </label>
```

Comment:
Currently this `label` isn't doing anything. The task list asks for "Each form field should also have a corresponding `<label>` element". That way the user knows what they're supposed to put in each of the inputs of the form. 

![](https://user-images.githubusercontent.com/9094098/193649697-6094afa7-ac22-4031-9d8b-a6d3ea4249fd.png)

Currently, your form looks like this:

With labels on _each_ of the inputs, it should look something like this:

![](https://user-images.githubusercontent.com/9094098/193650086-844d7df7-ff57-475c-ab2d-f93e46a9d96d.png)

Remember that each form control (`input`/ `textarea`, etc) should have _**both**_ an `id` attribute AND a `name` attribute. And the `label` element should have a `for` attribute that matches the `id` of the form control that it's labelling. You can either wrap the form control inside the `label` element or just put the label element before the form control:

```
<label for="email_input">Email: </label>
<input id="email_input" name="email" type="email" required>

OR 

<label for="email">Email:
  <input id="email_input" name="email" type="email" required>
</label>
```